### ISSUE TITLE
twint.run.favorites() returns nothing

### ISSUE THREAD

**Issue by BallM4788** [twintproject/twint#1085]
>Make sure you've checked the following:

- [X] Python version is 3.6;
- [X] Updated Twint with `pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint`;
- [X] I have searched the issues and there are no duplicates of this issue/question/request.

### Command Ran
```python
c = twint.Config()
c.Username = [USERNAME]
c.User_id = [USER_ID]
c.Format = "Username: {username} | TweetID: {id}"
c.Database = "recordedFaves.db"
c.Store_object = True
twint.run.Favorites(c)
```

### Description of Issue
No data is returned.
Console output:
```
[+] Inserting into Database: recordedFaves.db
list index out of range [x] feed.MobileFav
[!] Twitter does not return more data, scrape stops here.
```
Inserting "print(str(e))" at line 120 of run.py and repeating the above commands gives the same output with an additional line:
```
[+] Inserting into Database: recordedFaves.db
list index out of range [x] feed.MobileFav
RequestUrl() got an unexpected keyword argument 'headers'
[!] Twitter does not return more data, scrape stops here.
```

### Environment Details
Windows 7, in Eclipse via PyDev extension

**Comment by innocentius** [comment]
#1083 could be a similar issue.

**Comment by BallM4788** [comment]
Edited the issue to add new information.

**Comment by alexgeo20** [comment]
hi @BallM4788 have you had any luck scrapping favorites with this function?  I tried running it today and got the same error and I'm wondering if there is a fix.  Thank you!

`twint -u tim --favorites -o output.csv --csv

list index out of range [x] feed.MobileFav

[!] Twitter does not return more data, scrape stops here.`


cc @innocentius

**Comment by BallM4788** [comment]
Nope, still doesn't work. I've just resorted to using Twitter's API.

**Comment by Eleko** [comment]
hi, i have exactly the same issue with same 401 error.

**Comment by jnehring** [comment]
i get the same error using the command line:

```
$ twint -u AfD --favorites
list index out of range [x] feed.MobileFav
[!] Twitter does not return more data, scrape stops here.
```

there where some warnings in the shell output also which i ommited for better readibility. the warning was 4 times 

```
/home/jan/anaconda3/lib/python3.7/importlib/_bootstrap.py:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
```

**Comment by ShotoZhao** [comment]
i still have this issue, although refer to #870

**Comment by b0163ba** [comment]
twint -u USARNAME --favorites
list index out of range [x] feed.MobileFav
[!] Twitter does not return more data, scrape stops here.


What's the solution ???
Please.
